server/enforcer.go

```diff
@@ -89,6 +102,29 @@ func (s *Server) NewAdapter(ctx context.Context, in *pb.NewAdapterRequest) (*pb.
 	return &pb.NewAdapterReply{Handler: int32(h)}, nil
 }
 
+type ABACModel struct {
+	Name  string
+	Owner string
+}
```

Comment:
Sorry, I don't think that my comments were resolved - this model is still tied to a particular and a very narrow use case

Reply:
Maybe,  Do you have any good ideas?

Reply:
I listed them in the comments above. Do you think we can retrieve attributes in the `enforce` function from JSON objects just like we do from Go structs? That would eliminate the need to know the attributes beforehand or update them all the time. @hsluoyz

Reply:
It has been fixed.

Reply:
Do I understand correctly that now all attributes for objects must be called with names in the range from "v0" to "v10" and I can't use any other names?

Reply:
You can just use the names you want. ``v0`` to ``v10`` are just used internally and can be automatically converted to the right names.